client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+import { localize } from 'i18n-calypso';
+
+function redirectToPartner() {
+	if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+		window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
+	} else {
+		window.location.href = 'https://www.bluehost.com/wordpress';
+	}
+}
+
+function getPrice() {
+	return 'bluehost' === abtest( 'signupStoreBenchmarking' ) ? '$3.95' : '$12.95';
```

Comment:
I've discussed this with Meredith, and we concluded that it is fine to leave the prices in USD in order to keep them consistent with Bluehost and SiteGround, since they are not converting the amount based on locale. We will also postpone limiting to just USD users for now, and will iterate on that later if needed.